apps/connect/src/App.tsx

```diff
@@ -47,6 +50,19 @@ export default function Root() {
   useEffect(() => {
     localStorage.setItem("Connect Config", JSON.stringify(config, null, 2));
   }, [config]);
+
+  const Connect = (
+    <>
+      <WormholeConnect config={config} theme={customTheme} />
+      <Banner />
+    </>
+  );
+
+  const routes = useRoutes([
+    { path: `/`, element: Connect },
+    { path: `${USDCPath}`, element: Connect },
```

Comment:
please, double check this route, since we are deploying USDC Bridge as a separated app